queries.sql

```diff
@@ -36,3 +36,41 @@ SELECT neutered,MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-12-31' AND '2000-12-31' GROUP BY species;
 
+UPDATE animals SET species_id = 1 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE name NOT LIKE '%mon';
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon','Pikachu');
+UPDATE animals SET owner_id = 3 WHERE name IN ('Charmander','Squirtle','Blossom');
+UPDATE animals SET owner_id = 4 WHERE name IN ('Angemon','Boarmon');
+
+SELECT animals.name
+FROM animals
+JOIN owners ON owners.id = animals.owner_id 
+WHERE owners.full_name = 'Melody Pond';
+SELECT animals.name
+FROM animals
+JOIN species ON species.id = animals.species_id 
+WHERE species.name = 'Pokemon';
+SELECT owners.full_name ,animals.name
+FROM animals
+RIGHT JOIN owners ON owners.id = animals.owner_id;
```

Comment:
- [ ] When listing all the owners including the ones who do not have an animal, you are supposed to select from the owners table and then join the animals' table. Kindly update your query accordingly.